### DISCUSSION TITLE
is there a texstudio version supporting 32-bit?

### DISCUSSION THREAD

**Discussion by Gomesz785** [texstudio-org/texstudio#1705] [Q&A] [answered]
I tried the latest version, but it is not supported of 32-bit. so what is the newest texstudio version supporting 32-bit?

**Comment by sunderme** [comment, marked as answer]
2.12.22 , see https://github.com/texstudio-org/texstudio/releases/tag/2.12.22

**Reply by Gomesz785** [reply]
Thanks for the just-after-1-hour reply. Plese reply to my following question as it will give new users assurance..
So this version supports windows 32-bit?

**Reply by Gomesz785** [reply]
Self confirmation: it works.
